lms/services/html_service.py

```diff
@@ -0,0 +1,16 @@
+from html.parser import HTMLParser
+
+
+def strip_html_tags(html: str) -> str:
```

Comment:
This was a private function over the jstor service. 

Moved here and made public.